app/models/manageiq/providers/kubevirt/infra_manager/vm.rb

```diff
@@ -1,5 +1,6 @@
 class ManageIQ::Providers::Kubevirt::InfraManager::Vm < ManageIQ::Providers::InfraManager::Vm
   include Operations
+  include SupportsFeatureMixin
```

Comment:
This is already included in the base class (https://github.com/ManageIQ/manageiq/blob/master/app/models/vm_or_template.rb#L10) so you don't need to include it again
```suggestion
```